repository_service_tuf/cli/artifact/download.py

```diff
@@ -0,0 +1,266 @@
+# SPDX-FileCopyrightText: 2023 VMware Inc
+#
+# SPDX-License-Identifier: MIT
+
+
+import base64
+import os
+from hashlib import sha256
+from pathlib import Path
+from typing import Any, Dict, Optional
+from urllib import request
+
+import yaml
+from click import Context
+from tuf.api.exceptions import DownloadError, RepositoryError
+from tuf.ngclient import Updater
+
+from repository_service_tuf.cli import click, console
+from repository_service_tuf.cli.artifact import artifact
+from repository_service_tuf.cli.artifact.repository import find_hash_bin
+
+DOWNLOAD_DIR = "./downloads"
```

Comment:
IMO it shouldn't be nested, it should be the current dir `os. getcwd()` (multiplatform)